challenges/objects-arrays.js

```diff
@@ -7,23 +7,48 @@
 */
 
 // tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceious
+const tyrannosaurus = {
+diet: "carnivorous",
```

Comment:
These need to be indented, use the VS Code extension 'Prettier' to auto format on save.